projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html

```diff
@@ -122,6 +122,18 @@ <h5 translate>Patron Information</h5>
           {{ patron.patron.type.pid | getRecord: 'patron_types' : 'field' : 'name' | async }}
         </dd>
       </dl>
+      <!-- KEEP HISTORY -->
+      <dl class="row mb-0">
+        <dt class="col-sm-3 offset-sm-2 offset-md-0">
+          {{ 'Keep history' | translate }}:
```

Comment:
We are used to do like this in templates:
```
        <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>Keep history</dt>
```

`label-title` class will put the string in **bold** with a `:` at the end. And generally we use the directive rather than the pipe.